.gitignore

```diff
@@ -138,7 +138,13 @@ report.html
 howso/examples/output.csv
 
 # IDE files
-.vscode
+# VSCode
+.vscode/*
```

Comment:
This will cause local .vscode entries to have to be removed before pulling main if merged.